lib/src/migrators/division.dart

```diff
@@ -156,6 +122,44 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
         isDivisionAllowed: true);
   }
 
+  /// Migrates [node] and returns true if it is a new-syntax color function or
+  /// returns false if it is any other function.
+  bool _tryColorFunction(FunctionExpression node) {
+    if (!["rgb", "rgba", "hsl", "hsla"].contains(node.name.asPlain)) {
+      return false;
+    }
+    ListExpression channels;
+    if (node.arguments.positional.length == 1 &&
+        node.arguments.named.isEmpty &&
+        node.arguments.positional.first is ListExpression) {
+      channels = node.arguments.positional.first;
+    } else if (node.arguments.positional.isEmpty &&
+        node.arguments.named.containsKey(r'$channels') &&
+        node.arguments.named.length == 1 &&
+        node.arguments.named[r'$channels'] is ListExpression) {
+      channels = node.arguments.named[r'$channels'];
+    }
+    if (channels == null ||
+        channels.hasBrackets ||
+        channels.separator != ListSeparator.space ||
+        channels.contents.length != 3 ||
+        channels.contents.last is! BinaryOperationExpression) {
+      return false;
+    }
+    var last = channels.contents.last as BinaryOperationExpression;
+    if (last.left is! NumberExpression || last.right is! NumberExpression) {
+      _patchSpacesToCommas(channels);
```

Comment:
It's probably a good idea to provide an example of why this is necessary here. A really dedicated reader who's wondering what this is for could remove this code and see what tests fail, but it's better to save them the trouble.